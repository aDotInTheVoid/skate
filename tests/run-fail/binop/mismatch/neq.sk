fn main() {
    // TODO: Should this be different
    true != "x";
}