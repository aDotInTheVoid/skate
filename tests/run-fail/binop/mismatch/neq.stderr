error: Unknown binop `!=`, for `bool` and `string`
  ┌─ $DD/tests/run-fail/binop/mismatch/neq.sk:3:10
  │
3 │     true != "x";
  │     ---- ^^ --- LHS evaluated to "x"
  │     │    │   
  │     │    In this operator
  │     LHS evaluated to true

